mix.exs

```diff
@@ -29,11 +29,11 @@ defmodule Membrane.PortAudio.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
-      {:membrane_common_c, "~> 0.10.0"},
+      {:membrane_core, "~> 0.9.0"},
+      {:membrane_common_c, "~> 0.11.0"},
       {:bunch, "~> 1.3.0"},
```

Comment:
You can do the usual `mix.exs` cleanup
* [ ] `-` in license
* [ ] `formatters: ["html"]` in docs
* [ ] `mix deps.update --all`
* [ ] `mix deps.clean --unused`
* [ ] Bump versions of `ex_doc`, `dialyxir` & `credo`